### ISSUE TITLE
tts-server: add dropdown to select model

### ISSUE THREAD

**Issue by mariusa** [coqui-ai/TTS#737]
**Describe the bug**
In the docs screenshot there's a dropdown to select desired model:
https://github.com/coqui-ai/TTS/raw/main/images/demo_server.gif

but in local tts-server there's no dropdown:
![2021-08-17_00-00](https://user-images.githubusercontent.com/1283842/129628954-0f6c1424-ce02-4f80-99e0-8a3cd18e2182.png)

```
 tts --list_models
 Name format: type/language/dataset/model
 1: tts_models/en/ek1/tacotron2
 2: tts_models/en/ljspeech/tacotron2-DDC [already downloaded]
 3: tts_models/en/ljspeech/tacotron2-DDC_ph
 4: tts_models/en/ljspeech/glow-tts
 5: tts_models/en/ljspeech/tacotron2-DCA
 6: tts_models/en/ljspeech/speedy-speech-wn
 7: tts_models/en/ljspeech/vits
 8: tts_models/en/vctk/sc-glow-tts
 9: tts_models/en/vctk/vits
 10: tts_models/en/sam/tacotron-DDC
 11: tts_models/es/mai/tacotron2-DDC
 12: tts_models/fr/mai/tacotron2-DDC
 13: tts_models/zh-CN/baker/tacotron2-DDC-GST
 14: tts_models/nl/mai/tacotron2-DDC
 15: tts_models/de/thorsten/tacotron2-DCA
 16: tts_models/ja/kokoro/tacotron2-DDC
 17: vocoder_models/universal/libri-tts/wavegrad
 18: vocoder_models/universal/libri-tts/fullband-melgan
 19: vocoder_models/en/ek1/wavegrad
 20: vocoder_models/en/ljspeech/multiband-melgan
 21: vocoder_models/en/ljspeech/hifigan_v2 [already downloaded]
 22: vocoder_models/en/ljspeech/univnet
 23: vocoder_models/en/vctk/hifigan_v2
 24: vocoder_models/en/sam/hifigan_v2
 25: vocoder_models/nl/mai/parallel-wavegan
 26: vocoder_models/de/thorsten/wavegrad
 27: vocoder_models/de/thorsten/fullband-melgan
 28: vocoder_models/ja/kokoro/hifigan_v1
```

If the bug cause is having only 1 model downloaded, I'd expect for all models to be listed and download on demand.

**To Reproduce**

1. $ pip install TTS
2. $ tts-server
3. open provided URL in browser. Generating speech works, but there's no dropdown to select model.

**Expected behavior**
Dropdown to choose from all possible models, even if not downloaded yet.

**Environment (please complete the following information):**
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Fedora 34
- **PyTorch or TensorFlow version (use command below)**: ?
- **Python version**: Python 3.9.6
- **CUDA/cuDNN version**: ?
- **GPU model and memory**: ?
- **Exact command to reproduce**: above

**Additional context**
Since this is a local demo to try out Coqui, please have larger default limits. Now generated speech stops at 6 seconds
``` Decoder stopped with `max_decoder_steps` 500```

Please allow 10 min by default.

**Comment by erogol** [comment]
It is not a bug. It is the way it is intentionally not to bust people's computers with all these models. Maybe you can rephrase it to be a feature request in which I guess you want to download models on demand from the server UI.
